pom.xml

```diff
@@ -41,6 +41,10 @@
       <version>3.4.2</version>
     </dependency>
     <dependency>
+      <!--
+      Remove this in time since it's included in Paper but
+      this may be used elsewhere.
```

Comment:
The scope is test, so no if it's paper we can safely remove it

Reply:
That particular dependency isn't, it's compile

Reply:
Imagine actually reading properly